base/doc/usrguide.tex

```diff
@@ -837,10 +837,12 @@ \subsection{Using the verbatim argument types}
 
 Some additional details that may be useful for those with more \TeX{}
 knowledge: do not worry if this does not make sense to you! Spaces and tabs are
-stored as active characters. In Unicode engines, all other characters are of
-type \enquote{other}. In $8$-bit engines, the ASCII characters other than tab
-and space are of type \enquote{other}, and non-ASCII characters are active. As
-such, token-based comparisons are likely to fail unless set up properly.
+stored as active characters. In $8$-bit engines and non-ASCII characters are
```

Comment:
Sentense seems wrong:  In $8$-bit engines and non-ASCII characters

Reply:
Corrected